Example/Tests/Spec Helpers/SpecUtils.swift

```diff
@@ -89,7 +90,17 @@ struct SpecUtils {
         guid: GUID,
         status: AuthorizationStatus
     ) -> SEAuthorizationDataV2 {
-        return SEAuthorizationDataV2(id: "\(authorizationId)", connectionId: "\(connectionId)", status: status)
+        let dict: [String: Any] = [
+            "data": "",
+            "key": "",
+            "iv": "",
+            "id": authorizationId,
+            "connection_id": connectionId,
+            "status": "denied"
+        ]
+
+        let response = SpecDecodableModel<SEEncryptedAuthorizationData>.create(from: dict)
+        return (response?.decryptedAuthorizationDataV2!)!
```

Comment:
This one too. These methods are just helpers, there is no need to do it this way